app/controllers/group_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GroupController < ApplicationController
```

Comment:
Controller names should be pluralized. Instead of `GroupController` have `GroupsController`